### PR TITLE
Tests cypress ccpa

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

This improves the tests and allows ~~all~~ _some_ of them to run in CI. This is achieved with enforcing cookies.

- Better triggers for CCPA—now also looking at cookies set
- Upgrade to Cypress@5.3 &larr; 5.2
- Added notes for restrictions around CI, servers being located in the US, it’s impossible to run TCFv2 there.
- Update Svelte.app

## How to test

`yarn test:e2e:ui`

### CODE REVIEW

**Comment by mxdvl** [comment]
So it turns out we cannot enforce `gdprApplies`, as this is returned directly from Sourcepoint’s servers. Below is an example `curl` which changes the return value based on the client’s IP:

```zsh
curl -s $'https://sourcepoint.theguardian.com/wrapper/tcfv2/v1/gdpr/native-message\?requestUUID\=6e20cd3d-a9c0-43cf-af33-ce05d5038620\&hasCsp\=true\&env\=prod\&consentLanguage\=browserDefault\&body\=%7B%22accountId%22%3A1257%2C%22requestUUID%22%3A%226e20cd3d-a9c0-43cf-af33-ce05d5038620%22%2C%22propertyHref%22%3A%22https%3A%2F%2Ftest.theguardian.com%22%2C%22uuid%22%3A%22c359ee8e-96f2-4d53-bb48-2348b868738a%22%2C%22euconsent%22%3Anull%2C%22meta%22%3A%22%7B%5C%22mmsCookies%5C%22%3A%5B%5C%22_sp_v1_uid%3D1%3A417%3Af31f364d-4e28-4816-9dc8-9ecaca9f27a9%3B%5C%22%2C%5C%22_sp_v1_csv%3Dnull%3B%5C%22%2C%5C%22_sp_v1_lt%3D1%3A%3B%5C%22%2C%5C%22_sp_v1_opt%3D1%3Alogin%7Ctrue%3Alast_id%7C11%3A%3B%5C%22%2C%5C%22_sp_v1_data%3D2%3A171578%3A1601990557%3A0%3A2%3A0%3A2%3A0%3A0%3A_%3A-1%3B%5C%22%2C%5C%22_sp_v1_ss%3D1%3AH4sIAAAAAAAAAItWqo5RKimOUbKKRmbkgRgGtbE6MUqpIGZeaU4OkF0CVlBdi1tCSQduIFQKu7JYAAaOo-l5AAAA%3B%5C%22%2C%5C%22_sp_v1_consent%3D1\u00211%3A1%3A1%3A0%5C%22%5D%2C%5C%22resolved%5C%22%3Anull%7D%22%2C%22clientMMSOrigin%22%3A%22https%3A%2F%2Fsourcepoint.theguardian.com%22%2C%22targetingParams%22%3A%22%7B%5C%22framework%5C%22%3A%5C%22tcfv2%5C%22%7D%22%2C%22campaignEnv%22%3A%22prod%22%2C%22pubData%22%3A%7B%7D%7D' | cut -c1-256 | egrep '"gdprApplies":[a-z]+'
```

File: test-page/App.svelte [hunk 1]
```diff
@@ -20,6 +20,14 @@
...
 	function logEvent(event) {
 		eventsList = [...eventsList, event];
+		console.log(
+			`%c@guardian%c %cCMP%c [event]`,
+			'background: #052962; color: white; padding: 2px; border-radius:3px',
+			'',
+			'background: deeppink; color: white; padding: 2px; border-radius:3px',
+			'color: deepskyblue; ',
+			event,
+		);
```
Comment (mxdvl) [inline test-page/App.svelte:30]: Also added a slightly nicer console, based on @sndrs work.

File: cypress/integration/sourcepoint-tcfv2.spec.js [hunk 2]
```diff
@@ -25,28 +25,30 @@ describe('Window', () => {
 	});
 });
 
-// TODO: enable testing of TCFv2 on CI
-skipOn(Cypress.env('CI') === 'true', () => {
```
Comment (sndrs) [inline cypress/integration/sourcepoint-tcfv2.spec.js:29]: ❤️

**Review by sndrs — APPROVED** [review]
nice one!

**Pull request merged**